tdei-api-gateway.json

```diff
@@ -432,253 +468,208 @@
                                 }
                             }
                         }
                     },
+                    "400": {
+                        "description": "Bad request"
+                    },
                     "401": {
                         "description": "This request is unauthenticated."
                     },
                     "500": {
                         "description": "An server error occurred."
                     }
-                },
-                "security": [
-                    {
-                        "ApiKey": []
-                    },
-                    {
-                        "AuthorizationToken": []
-                    }
-                ]
-            },
+                }
+            }
+        },
+        "/api/v1/osw/confidence/calculate": {
             "post": {
                 "tags": [
-                    "GTFS-Flex"
+                    "OSW"
                 ],
-                "summary": "upload a new gtfs_flex file",
-                "description": "This call allows a user to upload or create a new gtfs flex file. The caller must provide metadata about the file. Required metadata includes information about how and when the data was collected and valid dates of the file. Returns the tdei_record_id of the uploaded file.",
-                "operationId": "uploadGtfsFlexFile",
+                "summary": "Initiate Confidence calculation for a dataset",
+                "description": "Initiates the confidence calculation of a tdeiRecordId as mentioned in the body",
+                "operationId": "oswConfidenceCalculate",
                 "requestBody": {
                     "content": {
-                        "multipart/form-data": {
+                        "application/json": {
                             "schema": {
-                                "required": [
-                                    "file",
-                                    "meta"
-                                ],
-                                "type": "object",
-                                "properties": {
-                                    "meta": {
-                                        "$ref": "#/components/schemas/GtfsFlexUpload"
-                                    },
-                                    "file": {
-                                        "type": "string",
-                                        "format": "binary"
-                                    }
-                                }
+                                "$ref": "#/components/schemas/OSWConfidenceRequest"
                             }
                         }
                     },
                     "required": true
                 },
                 "responses": {
-                    "202": {
-                        "description": "The request has been accepted for processing.  returns the tdei_record_id, unique identifier for uploaded file.",
+                    "200": {
```

Comment:
why is this 200, and not 202 like other async calls?

Reply:
@susrisha , fyi